consensus/ethash/sealer.go

```diff
@@ -46,18 +47,53 @@ var (
 	errInvalidSealResult = errors.New("invalid or stale proof-of-work solution")
 )
 
+// makeFakeDelay uses the ethash.threads value as a mean time (lambda)
+// for a Poisson distribution, returning a random value from
+// that discrete function. I think a Poisson distribution probably
+// fairly accurately models real world block times.
+// Note that this is a hacky way to use ethash.threads since
+// lower values will yield faster blocks, but it saves having
+// to add or modify any more code than necessary.
+func (ethash *Ethash) makeFakeDelay() float64 {
+	p := distuv.Poisson{
+		Lambda: float64(ethash.Threads()),
```

Comment:
Either should or should not install a randomness source here.

Reply:
@meowsbits Can you explain more as I don't understand what you want to be done here.

Reply:
The `distuv.Poisson` struct exports two fields: `Lamba` and `Src`. 

- https://pkg.go.dev/gonum.org/v1/gonum@v0.8.1/stat/distuv#Poisson
- https://pkg.go.dev/golang.org/x/exp/rand#Source

So we can set a pseudo-random source like `rand.NewSource(time.Now().UnixNano())`. Although I think there are far-edge cases where _not_ having a source would be desirable, I think they're easily outweighed by having one.

Reply:
Done with https://github.com/etclabscore/core-geth/pull/180/commits/e695a470652d0d73bbf75ddacb3e7d6725aea0c4